praw/models/reddit/removal_reasons.py

```diff
@@ -164,12 +204,12 @@ def add(self, message: str, title: str) -> RemovalReason:
 
         .. code-block:: python
 
-           reddit.subreddit('NAME').removal_reasons.mod.add(
+           reddit.subreddit('NAME').mod.removal_reasons.add(
                message='Foobar',
                title='Test')
 
         """
         data = {"message": message, "title": title}
         url = API_PATH["removal_reasons_list"].format(subreddit=self.subreddit)
-        reason_id = self.subreddit._reddit.post(url, data=data)
-        return RemovalReason(self._reddit, self.subreddit, reason_id)
+        id = self._reddit.post(url, data=data)
+        return RemovalReason(self._reddit, self.subreddit, id)
```

Comment:
No need to rename this variable (but the change from `self.subreddit._reddit` to `self._reddit` is good).